packages/lib/versionInfo.ts

```diff
@@ -32,9 +68,11 @@ export default function versionInfo(packageInfo: any) {
 		console.info(gitInfo);
```

Comment:
I have now removed this statement as part of the "no-console" rule that was just added. I assume it was safe to remove?

Reply:
I do not know the original intent of that line but it seems to me that it is safe to remove.